domains/aarya.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "aarya-16",
+        "email": "bodasaarya@gmail.com"
+    },
+    "record": {
+        "TXT": "7eb8cd46bba3d6d2f9a0fec57f0261"
```

Comment:
Is this some website verification or just random characters?

Reply:
It is the github pages verification

Reply:
thats not how you do that